dashboard/app/views/layouts/_header.html.haml

```diff
@@ -5,13 +5,19 @@
 - elsif browser.cdo_partially_supported?
   = render partial: 'layouts/partially_supported_browser'
 
-.header-wrapper.hide_on_mobile{class: ('feedback-bug-links' if @script_level && @script.show_report_bug_link?)}
-  - if @script_level && @script.show_report_bug_link?
+- show_bug_links = (@script_level && @script.show_report_bug_link?) || (@level && @level.is_project_level)
+
+.header-wrapper.hide_on_mobile{class: ('feedback-bug-links' if show_bug_links)}
+  - if show_bug_links
+    - report_url = @script_level ? @script_level.report_bug_url(request) : @level.report_bug_url(request)
     #betainfo
-      - if @script.feedback_url
-        = link_to t('landing.feedback'), @script.feedback_url
+      - if @script_level && @script.show_report_bug_link? && @script.feedback_url
+        = link_to t('landing.feedback'), @script.feedback_url, target: '_blank'
+        |
+      - elsif @level && @level.is_project_level && @level.game == Game.applab
+        = link_to t('landing.feedback'), 'https://www.surveymonkey.com/r/csp-applab', target: '_blank'
         |
-      = link_to t('landing.report_bug'), @script_level.report_bug_url(request), target: '_blank'
```

Comment:
We still call that on line 8 (if we have a @script_level) - which results in a slightly richer set of info being reported (script, stage, etc.)